dts: misc: Simplify the description of the binding

rruuaanng opened the pull request:
Remove redundant descriptions in MISC bindings, such as "This is a representation of" and "... node".

manuargue commented on dts/bindings/misc/nxp,s32-emios.yaml:
```diff
@@ -2,7 +2,7 @@
 
 description: |
-  NXP S32 Enhanced Modular IO SubSystem (eMIOS) node for S32 SoCs.
+  NXP S32 Enhanced Modular IO SubSystem (eMIOS) for S32 SoCs.
```
```suggestion
  Enhanced Modular IO SubSystem (eMIOS) for NXP S32 SoCs.
```
if you want to also rephrase this would be okay, same for the other `nxp,s32-*` bindings touched on this pr.

rruuaanng replied:
Looks good.

rruuaanng committed:
dts: misc: Simplify the description of the binding

Remove redundant descriptions in MISC bindings, such
as "This is a representation of" and "... node".

Signed-off-by: James Roy <rruuaanng@outlook.com>

manuargue reviewed (approved):
+1 NXP S32 changes
